version.c

```diff
@@ -44,6 +45,21 @@ void error(const char *format, ...)
     exit(-1);
 }
 
+void error_errno(const char *format, ...)
+{
+    va_list ap;
+    va_start(ap, format);
+    vfprintf(stderr, format, ap);
+    va_end(ap);
+    if (errno) {
+        fprintf(stderr, " : %s\n", strerror(errno));
```

Comment:
Need to sample `errno` before the `vfprintf` above, so might as well be the first thing in `error_errno()` before the `va_start` stuff too.

Perhaps no space before the colon (`": %s\n"`), depending on style in other htslib/bcftools error messages?

Reply:
Good spot; should be fixed now.